apps/live/src/controllers/health.controller.ts

```diff
@@ -0,0 +1,14 @@
+import type { Request, Response } from "express";
+import { Controller, Get } from "@plane/decorators";
+
+@Controller("/health")
+export class HealthController {
+  @Get("/")
+  async healthCheck(_req: Request, res: Response) {
+    res.status(200).json({
+      status: "OK",
+      timestamp: new Date().toISOString(),
+      version: process.env.APP_VERSION || "1.0.0",
+    });
```

Comment:
If this is health check is intended to be used by process managers to determine when to shutdown and provision replacement servers, then consider making this health check reflect the server's overall ability to perform it's intended functionality - e.g. it should check that it can reach outside dependencies like redis or anything else it depends on to serve requests. If connections to those services are down, this service is unhealthy and should not receive incoming traffic.